docs/explanations.rst

```diff
@@ -10,3 +10,4 @@ Explanation of how the library works and why it works that way.
 
     explanations/why-is-something-so
```

Comment:
Might as well delete this entry and the page it links to

Reply:
I did think about doing that :sweat_smile: